### ISSUE TITLE
[DataCap Application] <BLOCKCHAIN METAVERSE ACADEMY PTY LTD>

### ISSUE THREAD

**Issue by conneygabriele** [filecoin-project/filecoin-plus-large-datasets#1002]
# Large Dataset Notary Application

To apply for DataCap to onboard your dataset to Filecoin, please fill out the following.

## Core Information
- Organization Name: BLOCKCHAIN METAVERSE ACADEMY PTY LTD
- Website / Social Media:https://www.youtube.com/c/WorldBlockchainRobert/videos
- Total amount of DataCap being requested (between 500 TiB and 5 PiB):5PiB
- Weekly allocation of DataCap requested (usually between 1-100TiB):200TiB
- On-chain address for first allocation:f12k5kukgldua3ag5kzdhi27bxy52cvwf4kn7phfy

_Please respond to the questions below by replacing the text saying "Please answer here". Include as much detail as you can in your answer._

## Project details

Share a brief history of your project and organization.
```
Since 2016, I started to explain blockchain and have been engaged in blockchain education and training. I have uploaded more than 3,000 videos on major media platforms, and hundreds of paid course videos and paid live videos.
```

What is the primary source of funding for this project?
```
The revenue of the company.
```

What other projects/ecosystem stakeholders is this project associated with?
```
None.
```

## Use-case details

Describe the data being stored onto Filecoin
```
There are three main parts of the content that needs to be stored
1. What YouTube exposes
2. Exclusive to YouTube members
3. Teaching video (Shanghai local, Tencent classroom)
```

Where was the data in this dataset sourced from?
```
Video produced by myself.
```

Can you share a sample of the data? A link to a file, an image, a table, etc., are good ways to do this. 
```
https://www.youtube.com/c/WorldBlockchainRobert/videos
```
        
Confirm that this is a public dataset that can be retrieved by anyone on the Network (i.e., no specific permissions or access rights are required to view the data).
```
Yes.
```

What is the expected retrieval frequency for this data?
```
once every two months.
```

For how long do you plan to keep this dataset stored on Filecoin?
```
I hope it will be preserved forever, if not, it should be preserved for at least three years.
```

## DataCap allocation plan

In which geographies (countries, regions) do you plan on making storage deals?
```
The global.
```

How will you be distributing your data to storage providers? Is there an offline data transfer process?
```
We will distribute offline to storage providers.
```

How do you plan on choosing the storage providers with whom you will be making deals? This should include a plan to ensure the data is retrievable in the future both by you and others.
```
We will consider the stability of nodes and the time period of access to the network.
If SP can support fast retrieval, we will preferentially choose.
```

How will you be distributing deals across storage providers?
```
Try to distribute equally.
```

Do you have the resources/funding to start making deals as soon as you receive DataCap? What support from the community would help you onboard onto Filecoin?
```
Yes,We have prepared a special fund.
```

**Comment by raghavrmadya** [comment]
### Datacap Request Trigger
**Total DataCap requested**

> 5PiB

**Expected weekly DataCap usage rate**

> 200TiB

**Client address**

>f12k5kukgldua3ag5kzdhi27bxy52cvwf4kn7phfy

**Comment by raghavrmadya** [comment]
Additionally, if you can share the SPs you have been in touch with and the associated IDs, it will help gain community confidence

**Comment by dannyob** [comment]
I'd love to know how you get to the 5PiB estimate!

**Comment by conneygabriele** [comment]
Thanks for RG and Dannyob's attention and reply
About this dataset
It is divided into:
YouTube source video+material: 70TIB+
My Youtube account has been in operation for 4 years, with 51900 subscribers and 3938806 views
We uploaded about 1800 videos
In addition to Youtube, I have produced teaching videos on many social media platforms in China, including BiliBili, Youku, Tiktok, watermelon videos, etc

Shanghai International Community College: 300TIB+
We have set up a blockchain finance training school "Shanghai International Community College" in China and the Shanghai government of China. The college was established in December 2017. Robertli is the dean of the college. In the past five years, we have cooperated with Shanghai Jiaotong University, Fudan University, Tsinghua University, Shanghai University of Foreign Trade, Hong Kong Business School, Malaysia Client University and other universities to jointly launch courses and textbooks. It has trained senior talents in blockchain and digital economy, and promoted the research, development, application and popularization of blockchain technology.

Our Dataset Sources
518 blockchain salon (once a week for 6 years)
Blockchain Intermediate Qualification Certificate Training Course Video
Blockchain Digital Economy Direction Doctor of Business Administration (DBA) Course Video
Course related textbooks
Project research topic
To sum up, our existing data is about 400TIB. In addition to some sector fills, we plan to do 8 backups worldwide, so we need 5PIB quota.

Our monthly data increment is about 10TIB+
——————
The SPs we are currently in contact with include: Great heat, New web group, water drop cloud, Chain up cloud, F3, SXX and other Filecoin technology service companies.
In addition, we plan to find more and better SPs through BDE and Slack
Xi'an: f01907545
Shenzhen: f01099999, f01412203
Hong Kong: f01128206, f01885088, f01851482
Singapore: f01877571
Germany: f01926792
Australia: f01928022

**Comment by raghavrmadya** [comment]
8 copies of 400TiB adds up to 2 PiBs, not 5.PiBS

**Comment by raghavrmadya** [comment]
As there is a discrepancy in the information provided, this application is being closed. Please open a new application with the correct request as well as link this application

**Comment by raghavrmadya** [comment]
Please rectify the information in the LDN for notaries to proceed with due diligence

**Comment by conneygabriele** [comment]
Hi RG，
Sorry, I'm a little busy these days.
Our current dataset is 400TIB, but for the sake of data security and dispersion.
After communication with SP, we plan to fill 20G - 25G data in each 32G sector. The rest is filled with data.
This operation is compliant
So we need a copy of (400TIB * 1024)/(20G~25G) * 32G/1024 ≈ 640TIB~512TIB
8 replicas are required
640TIB ～ 512TIB *  8  = 5120TIB ～ 4096TIB 
Considering the growth of data sets in the packaging period, we hope to have 5PIB storage space.

**Comment by newwebgroup** [comment]
Robertli contacted us and we met through Zoom.
We reviewed the data samples and relevant qualifications, and we are willing to support them in the first round.

**Comment by newwebgroup** [comment]
## Request Proposed
Your Datacap Allocation Request has been proposed by the Notary
#### Message sent to Filecoin Network
>bafy2bzacedwg7wfruca7peo3z5oivzrcfoybbakp4emaeykdjidxnkqfj4xrs 
#### Address 
> f12k5kukgldua3ag5kzdhi27bxy52cvwf4kn7phfy
#### Datacap Allocated
> 100.00TiB
#### Signer Address
> f1e77zuityhvvw6u2t6tb5qlnsegy2s67qs4lbbbq
#### Id
> 
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacedwg7wfruca7peo3z5oivzrcfoybbakp4emaeykdjidxnkqfj4xrs

**Comment by xingjitansuo** [comment]
## Request Approved
Your Datacap Allocation Request has been approved by the Notary
#### Message sent to Filecoin Network
>bafy2bzacebw6tdxlh35gtuhgzk5nfabrrwnpsc7v5zbcwbuyfeneuc4londkg 
#### Address 
> f12k5kukgldua3ag5kzdhi27bxy52cvwf4kn7phfy
#### Datacap Allocated
> 100.00TiB
#### Signer Address
> f1a2lia2cwwekeubwo4nppt4v4vebxs2frozarz3q
#### Id
> 
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacebw6tdxlh35gtuhgzk5nfabrrwnpsc7v5zbcwbuyfeneuc4londkg

**Comment by xingjitansuo** [comment]
The material is convincing for now and I will check the distribution later on.

**Comment by BDE-io** [comment]
@conneygabriele Hi! Great to see you have gotten approval for DataCap and advancing the mission of preserving humanity’s most important information. If you are looking for storage providers to store these data or have any questions, please visit #bigdata-exchange on Filecoin Slack or reply here. 

We have strong demand from a diverse group of SPs, who are actively looking to onboard more data.

**Comment by kernelogic** [comment]
## Request Proposed
Your Datacap Allocation Request has been proposed by the Notary
#### Message sent to Filecoin Network
>bafy2bzacea54zxrf7syxo5gqrsprxafhmxql2cokkavbqzwgppaqzicpk4tzc 
#### Address 
> f12k5kukgldua3ag5kzdhi27bxy52cvwf4kn7phfy
#### Datacap Allocated
> 200.00TiB
#### Signer Address
> f1yjhnsoga2ccnepb7t3p3ov5fzom3syhsuinxexa
#### Id
> 43f50030-a755-4a4e-bc70-80ca548e9eb9
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacea54zxrf7syxo5gqrsprxafhmxql2cokkavbqzwgppaqzicpk4tzc

**Comment by YuanHeHK** [comment]
There is no problem with the number of SPs and data checks, but your data is too centralized, which is not in line with the filecoin principle. I will support you this time, but you need to adjust the subsequent packaging to ensure that the data is evenly distributed.

**Comment by YuanHeHK** [comment]
checker:manualTrigger

**Comment by YuanHeHK** [comment]
## Request Approved
Your Datacap Allocation Request has been approved by the Notary
#### Message sent to Filecoin Network
>bafy2bzaceckqxndthnwwvublk33bh7d3764s7ahslts6z3uaczzln6yqhngei 
#### Address 
> f12k5kukgldua3ag5kzdhi27bxy52cvwf4kn7phfy
#### Datacap Allocated
> 200.00TiB
#### Signer Address
> f1fg6jkxsr3twfnyhdlatmq36xca6sshptscds7xa
#### Id
> 43f50030-a755-4a4e-bc70-80ca548e9eb9
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzaceckqxndthnwwvublk33bh7d3764s7ahslts6z3uaczzln6yqhngei

**Comment by herrehesse** [comment]
Dear Applicant,

Due to the increased amount of erroneous/wrong Filecoin+ data recently, on behalf of the entire community, we feel compelled to go deeper into datacap requests. Hereby to ensure that the overall value of the Filecoin network and Filecoin+ program increases and is not abused.

Please answer the questions below as comprehensively as possible.

**Customer data**

- Could you demonstrate exactly how and to what extent customer contact occurred? 

We expect that for the onboarding of customers with the scale of an LDN there would have been at least multiple email and perhaps several chat conversations preceding it. A single email with an agreement does not qualify here.

- Did the customer specify the amount of data involved in this relevant correspondence?

- Why does the customer in question want to use the Filecoin+ program? 

_Should this only be soley for acquiring datacap this is of course **out of the question**. The customer must have a legitimate reason for wanting to use the Filecoin+ program which is intended as a program to store useful and public datasets on the network._ 

- Why is the customer data considered Filecoin+ eligible?

_(As an intermediate solution Filecoin offers the FIL-E program or the glif.io website for business datasets that do not meet the requirements for a Filecoin+ dataset)_


**Files and Processing**

- Could you please demonstrate to us how you envision processing and transporting the customer data in question to any location for preparation?
- Would you demonstrate to us that the customer, the preparer and the intended storage providers all have adequate bandwidth to process the set with its corresponding size?
- Would you tell us how the data set preparer takes into account the prevention of duplicates in order to prevent data cap abuse?


Hopefully you understand the caution the overall community has for onboarding the wrong data. We understand the increased need for Filecoin+, however, we must not allow the program to be misused. Everything depends on a valuable and useful network, let's do our best to make this happen. Together.

**Comment by NDLABS-OFFICE** [comment]
checker:manualTrigger

**Comment by NDLABS-OFFICE** [comment]
A client came to me on slack asking for a review of their LDN, and so far everything looks good, so I'm willing to support it. I hope that other members of the community can also conduct information review, and I will continue to pay attention.

**Comment by NDLABS-OFFICE** [comment]
## Request Proposed
Your Datacap Allocation Request has been proposed by the Notary
#### Message sent to Filecoin Network
>bafy2bzaceb5xihg4s3culxyt22wtzjwh6wqnxh2jxlomtcqymex635gsbpmhi 
#### Address 
> f12k5kukgldua3ag5kzdhi27bxy52cvwf4kn7phfy
#### Datacap Allocated
> 400.00TiB
#### Signer Address
> f1yayfsv6whu3rheviucvventj3y6t542xfpb47ei
#### Id
> 938f9520-a523-4b5c-82be-609b031ff651
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzaceb5xihg4s3culxyt22wtzjwh6wqnxh2jxlomtcqymex635gsbpmhi

**Comment by cryptowhizzard** [comment]
We are now 500 TiB on way with this datacap requests there are some questions to be raised here.

@conneygabriele i have some questions:

What software tools are you using to extract your Youtube video's or do you have a cold storage copy yourself?
What software tools have been used to prepare this dataset for distribution until now?
Are you distributing the dataset yourself or if not, who or which company is distributing it for you? 
How many TiB's of data have you packed already and do you have a list of CID's for us that are created?

Fil+ data should be stored in multiple regions. This data is stored only in the greater China region. Do you intent to store data on other continents according to the Fil+ guidelines?
You mention offline shipping of harddrives. How do you intent to ship that data to other continents?

Fil+ is a community program where transparency is key as it is incentivized by us all. I am looking forward to your answers.

**Comment by stcouldlisa** [comment]
## Request Approved
Your Datacap Allocation Request has been approved by the Notary
#### Message sent to Filecoin Network
>bafy2bzacebo2r6g7asdafhqthzajikbh5as2cygxptzed7sxwj4uisxqyc2bq 
#### Address 
> f12k5kukgldua3ag5kzdhi27bxy52cvwf4kn7phfy
#### Datacap Allocated
> 400.00TiB
#### Signer Address
> f1jvvltduw35u6inn5tr4nfualyd42bh3vjtylgci
#### Id
> 938f9520-a523-4b5c-82be-609b031ff651
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacebo2r6g7asdafhqthzajikbh5as2cygxptzed7sxwj4uisxqyc2bq

**Comment by stcouldlisa** [comment]
<img width="770" alt="WX20230106-184142@2x" src="https://user-images.githubusercontent.com/112812896/210994345-a025c0c3-b593-416b-824e-ee435ba7d00a.png">

**Comment by stcouldlisa** [comment]
<img width="1144" alt="WX20230106-184340@2x" src="https://user-images.githubusercontent.com/112812896/210994562-848ce85f-5b26-4240-ac73-278bbb5faf96.png">

**Comment by cryptowhizzard** [comment]
@stcouldlisa no interest in any of the questions asked right?

**Comment by herrehesse** [comment]
@conneygabriele the data is highly centralized and stored in the same region, which is against the rules of the Filecoin+ program.  There is also 91.62% of deals are for data replicated across less than 4 storage providers.

Can you explain how you will resolve this and why you did not follow the guidelines of the program?

@stcouldlisa - can you explain why you signed without due diligence?

**Comment by stcouldlisa** [comment]
@herrehesse Sorry for the late reply, I signed because the client told me that he has already answered your question, and also contacted you or your colleagues on slack, I saw the client's explanation, I think the client's The explanation is reasonable.

**Comment by stcouldlisa** [comment]
<img width="936" alt="WX20230109-094130@2x" src="https://user-images.githubusercontent.com/112812896/211229694-0b1a2ad3-e683-48bf-89cb-f2085c3bded0.png">

**Comment by stcouldlisa** [comment]
Of course, I think @herrehesse should find out the reason why he cannot log in to GitHub as soon as possible, and then reply to your question on GitHub,

**Comment by stcouldlisa** [comment]
Of course, I think @conneygabriele  should find out the reason why he cannot log in to GitHub as soon as possible, and then reply to your question on GitHub,

**Comment by Tom-OriginStorage** [comment]
checker:manualTrigger

**Comment by Tom-OriginStorage** [comment]
hello, Please explain that there is a SP packaging ratio of nearly 50%?

**Comment by Tom-OriginStorage** [comment]
Can you answer Herrehesse's question?

**Comment by Tom-OriginStorage** [comment]
![image](https://user-images.githubusercontent.com/103015469/211235463-10bb7c8e-8f62-419a-82b2-def64499302b.png)

**Comment by Tom-OriginStorage** [comment]
![image](https://user-images.githubusercontent.com/103015469/211235501-224db0a9-1ec4-495c-ac8f-9fcf423cf8cb.png)

**Comment by Tom-OriginStorage** [comment]
![image](https://user-images.githubusercontent.com/103015469/211235523-fb998351-eb74-4ab5-aa05-f28ba540a2a7.png)

**Comment by Tom-OriginStorage** [comment]
## Request Proposed
Your Datacap Allocation Request has been proposed by the Notary
#### Message sent to Filecoin Network
>bafy2bzacecle7xtccyzsmetojqfjdprlpvjmwrgvlgfn4x4pqftisw3ymdurw 
#### Address 
> f12k5kukgldua3ag5kzdhi27bxy52cvwf4kn7phfy
#### Datacap Allocated
> 800.00TiB
#### Signer Address
> f1q6bpjlqia6iemqbrdaxr2uehrhpvoju3qh4lpga
#### Id
> 42bb9c17-a34f-4e1f-a75e-2e92e7ac5dd7
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacecle7xtccyzsmetojqfjdprlpvjmwrgvlgfn4x4pqftisw3ymdurw

**Comment by newwebgroup** [comment]
Check the history of Cid checker and GIthub, as well as the screenshot of Slack chat.It seems all right.
We are willing to support this round and look forward to new SPs joining in 10 days

**Comment by newwebgroup** [comment]
## Request Approved
Your Datacap Allocation Request has been approved by the Notary
#### Message sent to Filecoin Network
>bafy2bzacedxre25b7shh3yo63rmx7l5qjhlv4b6zq72wpch2lhjuo52hbw3c6 
#### Address 
> f12k5kukgldua3ag5kzdhi27bxy52cvwf4kn7phfy
#### Datacap Allocated
> 800.00TiB
#### Signer Address
> f1e77zuityhvvw6u2t6tb5qlnsegy2s67qs4lbbbq
#### Id
> 42bb9c17-a34f-4e1f-a75e-2e92e7ac5dd7
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacedxre25b7shh3yo63rmx7l5qjhlv4b6zq72wpch2lhjuo52hbw3c6

**Comment by herrehesse** [comment]
I hereby request a immediate dispute on the notary @newwebgroup. The approval of this datacap without proper due diligence is against all of the Filecoin+ rules and community guidelines.

I highly recommend no notaries to sign this request until further notice.

@stcouldlisa @Tom-OriginStorage the given answers are clearly not enough and demand a follow up. The movements of @newwebgroup show again how to not behave as a notary.

**Comment by BDEio** [comment]
@conneygabriele Hi! Great to see that you have gotten approval for DataCap!
BDE is a verified deals auction house helping you to get paid storing your data with reliable storage providers. If you need any help, please get in touch.
